### PR TITLE
[new] Document new password change flow

### PR DESCRIPTION
**Pull request by angelaLazar**
I was not able to test this locally to see if the links, variables, and
the HTTP Request were working properly.
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [x] Commit messages conforms to our [commit message standards](https://github.com/auth0/docs#commit-messages)
- [x] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [ ] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 
### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

File: articles/connections/database/password-change.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+A password reset email can be sent to a user using the API.  Users themselves can also reset their own password on the login screen.
+
+Note that passwords can only be changed for users signing in using database connections. If a user is signing in with a social or enterprise connection, their password would need to be reset in those systems.
+
+## Using the API
+
+When calling the API, specify the email address of the account you want to reset the password for in the `email` field. A successful result of this request will send an email to the specified user for a password change request.
+
+{
```
Comment (ntotten) [inline articles/connections/database/password-change.md:22]: code blocks need to be surrounded in ```

**Comment by angelaLazar** [comment]
fixed the code formatting

File: articles/connections/database/password-change.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Note that passwords can only be changed for users signing in using database connections. If a user is signing in with a social or enterprise connection, their password would need to be reset in those systems.
+
+## Using the API
+
+When calling the API, specify the email address of the account you want to reset the password for in the `email` field. A successful result of this request will send an email to the specified user for a password change request.
+
+```
+{
+    "method": "POST",
```
Comment (ntotten) [inline articles/connections/database/password-change.md]: indentation in code should always be 2 spaces

File: articles/connections/database/password-change.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+[](/media/articles/connections/database/reset-password.png)
+
+Note that this email link is only valid once and only for the amount of time specified in the `URL Lifetime` field. This field can be changed where you customize the change password email.
+
+[Click here for more details on the dbconnections/change_password API call](auth-api#!#post--dbconnections-change_password)
+
+## Customizing the Change Password Email
+
+You can change the content of the password change request emails in the  [Emails > Templates](${uiURL/#/emails}) section of the dashboard.
```
Comment (ntotten) [inline articles/connections/database/password-change.md]: This should be `${uiURL}/#/emails`

File: articles/connections/database/password-change.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+[](/media/articles/connections/database/change-password-email.png)
+
+## Manually Setting a User's Password
+
+::: panel-danger Notice
+Changing a user's password in this way does not alert them to a password change.
+:::
+
+A user's password can be changed manually in the [Users](${uiURL/#/users}) section of the dashboard.
```
Comment (ntotten) [inline articles/connections/database/password-change.md]: same `${uiURL}/#/users`

File: articles/connections/database/password-change.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+---
+url: /password_change
+---
+
+# Changing a User's Password
+
```
Comment (ntotten) [inline articles/connections/database/password-change.md:6]: Myabe add a sentence like "this document explains the various ways you can reset passwords..."

**Pull request merged**